Public/Src/Engine/UnitTests/FingerprintStore/FingerprintStoreTests.cs

```diff
@@ -149,7 +149,7 @@ public void DontOverwriteExistingContentAddressableEntries()
 
             var counters = testHooks.FingerprintStoreTestHooks.Counters;
             XAssert.AreEqual(1, counters.GetCounterValue(FingerprintStoreCounters.NumPathSetEntriesPut));
-            XAssert.AreEqual(1, counters.GetCounterValue(FingerprintStoreCounters.NumDirectoryMembershipEntriesPut));
+            XAssert.IsTrue(counters.GetCounterValue(FingerprintStoreCounters.NumDirectoryMembershipEntriesPut) >= 1);

```

Comment:
I don't know what this test is exactly supposed to do, but it's called "DontOverwriteExisting...Entries", which makes me think that checking `NumDirectoryMembershipEntriesPut == 1` vs `NumDirectoryMembershipEntriesPut >= 1` is significant